### PR TITLE
Bob 13 create placable tower

### PR DESCRIPTION
**Pull request by DonTars**
you can place a tower after collection 5 wood (resources and count set within the prefab)

### CODE REVIEW

File: Assets/Editor/Tests/Help/HelperClass.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+public class HelperClass
+{
+    public static void WaitForSeconds(float seconds)
+    {
+        var start = Time.time;
+        while (Time.time < start + seconds)
+        {
+        }
+    }
+}
```
Comment (SomethingWittyWithATechAspect) [inline Assets/Editor/Tests/Help/HelperClass.cs:12]: end of file new line

Comment (DonTars) [reply Assets/Editor/Tests/Help/HelperClass.cs:12]: done

File: Assets/Editor/Tests/AutoShooterTest.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        {
+            _altUnityDriver.FindObject(By.NAME, "Enemy");
+            Assert.IsTrue(false);
+        }
+        catch
+        {
+            // ignored
+        }
+    }
+}
```
[the remark this answers is not in the document]
Comment (DonTars) [reply Assets/Editor/Tests/AutoShooterTest.cs]: done

**Comment by SomethingWittyWithATechAspect** [comment]
way to fast for me

**Comment by SomethingWittyWithATechAspect** [comment]
Tag tower missing in Tags and Layer file. Is it missing from commit

**Review by SomethingWittyWithATechAspect — APPROVED** [review]

**Comment by DonTars** [comment]
fixes look good for me, merging

**Pull request merged**